### PR TITLE
fix: typos in README

### PR DESCRIPTION
**Pull request by pointmax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ With AFFiNE, every block group has infinite views, for you to keep your single s
 We would like to give special thanks to the innovators and pioneers who greatly inspired us:
```
[the remark this answers is not in the document]
Comment (chrishollandaise) [reply README.md:142]: Also, I believe like there should be pluralization of the word "database" and an definite article before the word "Monday."

**Before:** We really appreciate the idea of Monday, Airtable and Notion database.
**After:** We really appreciate the idea of the Monday, Airtable and Notion databases.

**Review by chrishollandaise — COMMENTED** [review]
Hey pointmax! Good work on the finding theses typos. I marked some more typos that you missed that you can fix and roll into this PR.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (HeJiachen-PM) [reply README.md:142]: Hello! Thank you for your great work on the check. Would you consider opening another PR for these typos?

**Review by HeJiachen-PM — COMMENTED** [review]
Great work on fixing the typos.

**Comment by HeJiachen-PM** [comment]
Hi guys, many thanks for perfecting the README. I will just merge them as soon as the conversations are resolved.

**Comment by pointmax** [comment]
Hey thanks @chrishollandaise, I have fixed the typos you have mentioned

**Review by HeJiachen-PM — APPROVED** [review]

**Pull request merged**